website/blog/2021-11-09-blog1.md

```diff
@@ -0,0 +1,48 @@
+---
+slug: Blog 1
+title: WARP Whole Genome and Exome Pipelines Produce Reblocked GVCFs
+author: Liz Kiernan
+author_title: Senior Science Writer, Data Sciences Platform
+author_image_url: https://avatars.githubusercontent.com/u/14366016?v=4
+tags: [warp, WDL]
```

Comment:
genomes, exomes, germline, joint calling

Maybe those tags should be added? I know we haven't blogged much lol but seems forward looking and representative.

Reply:
Good idea! I added them.